mgr/Conscript-standard

```diff
@@ -908,26 +908,18 @@ if ( $pkg !~ /^sim$/ && $pkg !~ /^gen$/ ) {
 	$CPPPATH = "#StarVMC" . $main::PATH_SEPARATOR . $CPPPATH;
 	#    print "StarVMC change CPPPATH to $CPPPATH =========\n";
 
-    } elsif ($pkg eq "TPCCATracker" or $pkg eq "StiCA") {
+    }  elsif ( $pkg eq 'TPCCATracker' or $pkg eq 'TPCCATrackerPerformance' or $pkg eq 'StiCA') {
         $CPPPATH .=
-	  $main::PATH_SEPARATOR . "#" . $Dir . $main::PATH_SEPARATOR . "#" . $Dir . "/code";
-
-        if ( defined($ENV{Vc_DIR}) ) {
-          $CPPPATH = $ENV{Vc_DIR}."/include".$main::PATH_SEPARATOR.$CPPPATH;
-          $LIBPATH = $ENV{Vc_DIR}."/lib";
-        }
-
-	my $extraCXXFLAGS = " -DNVALGRIND -DENABLE_VECTORIZATION -W -Wall -Wswitch -Wformat -Wchar-subscripts";
-	$extraCXXFLAGS .= " -Wparentheses -Wmultichar -Wpointer-arith -Wcast-align -Wreturn-type -Wno-unused-function";
-	$extraCXXFLAGS .= " -pedantic -Wno-long-long -Wno-variadic-macros -msse -msse2 -Wshadow -fno-threadsafe-statics -DTBB_DO_ASSERT=1";
-	if ($STAR_SYS eq 'sl64_gcc447' or $STAR_SYS eq 'sl64_x8664_gcc447') {$extraCXXFLAGS .= " --param large-function-growth=500";}
-	if ($STAR_SYS eq 'sl64_gcc447' or $STAR_SYS eq 'sl64_x8664_gcc447') {$extraCXXFLAGS .= " --param large-function-growth=500";}
-	$extraCXXFLAGS .= " -DNODEBUG";
-	$CXXFLAGS .= $extraCXXFLAGS;
-	$CXXFLAGS .= $env->{Packages}->{Vc}->{CPP};
-	if ( defined($ENV{Vc_LIB_DIR}) ) { die("Vc_LIB_DIR is not supported anymore. Please use Vc_DIR."); }
-	$LIBS = "-lVc";
-	if ($extraCXXFLAGS =~ /USE_TBB/) {$LIBS .= " -lTBB";}
+            $main::PATH_SEPARATOR . "#" . $Dir . "/../TPCCATrackerPerformance"
+            . $main::PATH_SEPARATOR . "#" . $Dir . "/../TPCCATracker";
+        $extraCXXFLAGS = " -DNODEBUG -DNVALGRIND ";
+        $extraCXXFLAGS .= " -Wno-parentheses -Wno-unused-function -Wno-deprecated-declarations -Wno-unused-but-set-variable -Wno-unused-variable";
```

Comment:
Why are you disabling warnings for unused but set variables? Wouldn't it be the right thing to not declare variables if they are unused?

Reply:
That is for TPCCATracker. They keep some variables there for the convinions to do tuing. To clear up, we need manpower frome the GSI group.

Reply:
Since you like consistency... let's try to keep the same compiler flags as for all other packages unless we really have to add or modify them. The reasons are:

- Minimize the number of changes
- Less options to worry about in case we switch to a new version/compiler

Reply:
It is not inconsistency here. These flags are recommended by the CA tracker developers. Once we have new version/compiler, they should review these flags.